cloudevents-sdk-warp/src/lib.rs

```diff
@@ -0,0 +1,254 @@
+#[macro_use]
+mod headers;
+
+mod server_request;
+mod server_response;
+
+pub mod reply {
+    use crate::server_response::event_to_response;
+
+    use cloudevents::Event;
+    use http::StatusCode;
+    use warp::reply::Response;
+
+    ///
+    /// # Serializes `CE` as a http response
```

Comment:
Never call `CE` a cloudevent, always use the full name `CloudEvent`. In this case, since you're referring to the `Event` struct, use the link notation to generate the internal link:

```
[`Event`]
```

Check it in other doc comments too